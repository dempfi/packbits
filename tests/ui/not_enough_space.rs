use packbits as _;

#[packbits::pack(bytes = 1)]
struct Bad {
  a: u8,
  #[bits(1)]
  b: u8,
}

fn main() {}
