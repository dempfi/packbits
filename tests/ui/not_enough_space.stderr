error: not enough space for field
 --> tests/ui/not_enough_space.rs:6:3
  |
6 |   #[bits(1)]
  |   ^
